config/copycat-standalone.properties

```diff
@@ -16,12 +16,21 @@
 # These are defaults. This file just demonstrates how to override some settings.
 bootstrap.servers=localhost:9092
 
+# The converters specify the format of data in Kafka and how to translate it into Copycat data. Every Copycat user will
+# need to configure these based on the format they want their data in when loaded from or stored into Kafka
 key.converter=org.apache.kafka.copycat.json.JsonConverter
```

Comment:
Right. I missed that. 

We can discuss, but I have to admit that I can't think of anything better either. 

We may want to document (for users) what this configuration mean / does.